polaris-core/src/test/java/org/apache/polaris/core/storage/azure/AzureCredentialsStorageIntegrationTest.java

```diff
@@ -32,20 +32,20 @@ public class AzureCredentialsStorageIntegrationTest {
   public void testAzureCredentialFormatting() {
     Instant expiresAt = Instant.ofEpochMilli(Long.MAX_VALUE);
 
-    AccessConfig noSuffixResult = toAccessConfig("sasToken", "some_account", expiresAt);
+    AccessConfig noSuffixResult = toAccessConfig("sasToken", "some_account", expiresAt, null);
```

Comment:
Could you add tests for non-null refresh endpoint values? (sorry, if I missed them).

Reply:
I changed one of the tests to use a non null value already, I'll see if there are more places worth doing it.